zulipterminal/ui_tools/views.py

```diff
@@ -293,13 +293,17 @@ def __init__(self) -> None:
 class StreamsView(urwid.Frame):
     def __init__(self, streams_btn_list: List[Any], view: Any) -> None:
         self.view = view
-        self.log = urwid.SimpleFocusListWalker(streams_btn_list)
+        # Create Stream List Box
         self.streams_btn_list = streams_btn_list
-        self.focus_index_before_search = 0
+        self.log = urwid.SimpleFocusListWalker(streams_btn_list)
```

Comment:
Could you explain your reasoning behind the change? I don't think it is necessary.

Reply:
The exchange was done to facilitate the grouping